CONSTANTS.py

```diff
@@ -0,0 +1 @@
+KEY = "BccKLSpQGIdcP5CYpEQ-nPEQhMuNmFMguTkf7gyyIf0="
```

Comment:
If you have only one constant then you can move the constant to the `main.py` module.